src/main/java/org/sonar/squidbridge/annotations/AnnotationBasedProfileBuilder.java

```diff
@@ -27,12 +27,13 @@
 import org.sonar.api.utils.ValidationMessages;
 
 /**
+ * @deprecated since 2.7 Profile should be manually created.
```

Comment:
Suggestion: "Plugins should define quality profiles with org.sonar.api.server.profile.BuiltInQualityProfilesDefinition"